### PR TITLE
Ajax

### PR DESCRIPTION
**Pull request by zoebarth**
Use ajax and jquery to add new comments without refreshing page or redirecting user, move selection element for aesthetic purposes.

Feel free to test functionality here:
https://zoebarth-step-2020.appspot.com/#contact-me

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -170,30 +170,30 @@ <h1>Contact Me</h1>
...
           <option value="all">All</option>
         </select>
+        <div id="comments-container"> </div>
         <br />
         <button class="btn btn-danger btn-sm" onclick="deleteComments()">Delete all comments</button>
       </div>
     </section>
 
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="script.js"></script>
```
Comment (genedwards) [inline portfolio/src/main/webapp/index.html:193]: you already have this in the header, remove this duplicate

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -170,30 +170,30 @@ <h1>Contact Me</h1>
...
           <option value="25">25</option>
           <option value="all">All</option>
         </select>
+        <div id="comments-container"> </div>
         <br />
         <button class="btn btn-danger btn-sm" onclick="deleteComments()">Delete all comments</button>
       </div>
     </section>
 
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (genedwards) [inline portfolio/src/main/webapp/index.html:192]: I'm confused, this looks like you have 2 copies of jquery, jquery 3.2.1 & jquery 3.5.1. Do you need to load both? and if so, why?

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -70,3 +70,16 @@ async function deleteComments() {
...
   messagesElement.innerHTML = '';
 }
+
+function handleSubmit(e) {
+  e.preventDefault();
+  $.ajax({
+    type: 'POST',
+    url: '/data',
+    data: $(this).serialize(),
+    success: () => fetchContent(),
```
Comment (genedwards) [inline portfolio/src/main/webapp/script.js:80]: If you're not passing arguments, you should be able to do a shorthand `success: fetchContent,`

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -70,3 +70,16 @@ async function deleteComments() {
...
   messagesElement.innerHTML = '';
 }
+
+function handleSubmit(e) {
+  e.preventDefault();
+  $.ajax({
+    type: 'POST',
+    url: '/data',
+    data: $(this).serialize(),
+    success: fetchContent(),
```
Comment (genedwards) [inline portfolio/src/main/webapp/script.js:80]: are you sure this works? it should be `success: fetchContent,`

Comment (zoebarth) [reply portfolio/src/main/webapp/script.js:80]: Ah you're right, thank you!

**Pull request merged**